### DISCUSSION TITLE
Does slideflow permit research use in industry?

### DISCUSSION THREAD

**Discussion by Meijian** [jamesdolezal/slideflow#295] [Q&A] [answered]
Hi @jamesdolezal, I'm from a pharma company and I'd like to use slideflow for research work. Some of the work might be published in scientific journals in the future. In your GitHub readme page, you stated the following:
**License
This code is made available under the GPLv3 License and is available for non-commercial academic purposes.**

My question is, do you see any problems with me using this tool? We are not profiting off it of course.

Thanks,
Meijian

**Comment by jamesdolezal** [comment, marked as answer]
Hi Meijian,

Thanks for raising this question. Internally, we have been discussing Slideflow's licensing and considering the idea of opening it up further (e.g. MIT) to make industry collaboration easier (and potentially allowing commercial use). 

Under our current license, you are free to use Slideflow for academic work, whether in industry or an academic institution, as long as the code is not used in a commercial product and derivative work is also open-source. We would also appreciate a citation, but this is not strictly required by the license. 

If others have thoughts on Slideflow's licensing and the possibility of opening it up for commercial use, we can continue the discussion here. Or, if you would prefer, please also feel free to email me directly (james.dolezal@uchicagomedicine.org).

**Reply by Meijian** [reply]
Hi James, really appreciate your response. It's good to know that I can use it in the industry too for academic work. I will make sure to cite your paper when publishing the work.

Thanks,
Meijian
